pandas/io/tests/test_packers.py

```diff
@@ -341,6 +342,7 @@ def setUp(self):
         self.d['mixed'] = Series(data['E'])
         self.d['dt_tz_mixed'] = Series(data['F'])
         self.d['dt_tz'] = Series(data['G'])
+        self.d['categorical'] = Series(data['H'])
```

Comment:
It wouldn't make a difference here, e.g. `df['c'] = Categorical(..)` and `df['c'] = Series(Categorical(..))` do the same thing